packages/lib/services/interop/InteropService_Importer_Md.ts

```diff
@@ -47,13 +47,16 @@ export default class InteropService_Importer_Md extends InteropService_Importer_
 
 	async importDirectory(dirPath: string, parentFolderId: string) {
 		console.info(`Import: ${dirPath}`);
-
 		const supportedFileExtension = this.metadata().fileExtensions;
 		const stats = await shim.fsDriver().readDirStats(dirPath);
 		for (let i = 0; i < stats.length; i++) {
 			const stat = stats[i];
 
 			if (stat.isDirectory()) {
+				if (await this.isDirectoryEmpty(`${dirPath}/${basename(stat.path)}`)) {
```

Comment:
Why "basename"? It should be `dirPath + stat.path` or the logic is not right.

Reply:
Yes, it should be `dirPath + stat.path`. It is a mistake.